src/Components/ImportModal/ImportModal.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { ViewMedia, DragModal } from "./ViewMedia";
+import styles from "./ImportModal.module.css";
+import { Button } from "react-bootstrap";
+
+type Props = {
+  setShow: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Are you sure this is the type required by TS, and you can't just declare a function type instead? It would read cleaner if this could just be `setShow: (show: boolean) => void` or similar